R/rdeephaven/R/client_wrapper.R

```diff
@@ -138,22 +179,46 @@ Client <- R6Class("Client",
         client_options = options
       )
     },
+
+    #' @description
+    #' Creates an empty table on the server with 'size' rows and no columns.
+    #' @param size Non-negative integer specifying the number of rows for the new table.
+    #' @return TableHandle reference to the new table.
     empty_table = function(size) {
       verify_nonnegative_int("size", size, TRUE)
       return(TableHandle$new(self$.internal_rcpp_object$empty_table(size)))
     },
+
+    #' @description
+    #' Creates a ticking table on the server.
+    #' @param period ISO-8601-formatted string specifying the update frequency of the new table.
+    #' @param start_time Optional ISO-8601-formatted string specifying the start time of the table.
+    #' Defaults to now.
+    #' @return TableHandle reference to the new table.
     time_table = function(period, start_time = "now") {
       verify_string("period", period, TRUE)
       verify_string("start_time", start_time, TRUE)
       return(TableHandle$new(self$.internal_rcpp_object$time_table(period, start_time)))
     },
+
+    #' @description
+    #' Opens a table named `name` from the server if it exists.
```

Comment:
```suggestion
    #' Gets a reference to a table on the server using a variable name in the server's scope.
```
More or less clear?

Reply:
I don't find either as clear as it could be. Maybe "Gets a reference to a table named 'name' on the server." ?

Reply:
I'm concerned that they view 'name' as the variable name.